src/System.Management.Automation/engine/remoting/fanin/PSPrincipal.cs

```diff
@@ -126,17 +125,15 @@ public PSSenderInfo(PSPrincipal userPrincipal, string httpUrl)
             // cmdlets/scripts a chance to modify these.
         }
 
-#if !CORECLR // TimeZone Not In CoreCLR
         /// <summary>
         /// Contains the TimeZone information from the client machine.
         /// </summary>
-        public TimeZone ClientTimeZone
+        public TimeZoneInfo ClientTimeZone
         {
             get { return clientTimeZone; }
             internal set { clientTimeZone = value; }
         }
-        private TimeZone clientTimeZone;
-#endif
+        private TimeZoneInfo clientTimeZone;
```

Comment:
Minor issue.  We are trying to move away from explicit field members for properties and instead to this:
```powershell
public TimeZoneInfo ClientTimeZone
{
    get;
    internal set;
}
```
Can you please remove the private clientTimeZone field?